.github/workflows/ci.yml

```diff
@@ -21,9 +21,9 @@ jobs:
       uses: actions/cache@v1
       with:
         path: ./vendor/bundle
-        key: v1-linux-2.3
+        key: v1-linux-2.3-{{ checksum rmagick.gemspec }}
```

Comment:
@Watson1978 why did you need to add `checksum` here?

Reply:
To drop cache and install new version, it is useful to check changing its file.

Reply:
Doesn't `bundle` install any missing dependencies either way? I think this might slow down the build any time `gemspec` changes.